src/Take.Blip.Client/Content/InputExpirationTimeDocument.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Lime.Protocol;
+using System.Runtime.Serialization;
+
+namespace Take.Blip.Client.Content
+{
+    /// <summary>
+    /// Represents an expired input with an user's <see cref="Identity"/> to make a fake input
+    /// </summary>
+    [DataContract]
+    public class InputExpirationTimeDocument : Document
```

Comment:
This suffix `Document` is not used on any other Document type, it's better drop it. 
Maybe exchange for something like 'Marker' could make it more clear too.